kratos/sources/gid_io.cpp

```diff
@@ -0,0 +1,25 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Miguel Angel Celigueta
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/gid_io.h"
+
+namespace Kratos
+{
+    int GiDIOBase::msLiveInstances = 0;
```

Comment:
why does this need to be in the cpp?

Reply:
Hi @philbucher, I did not manage to add it in the .h file. As it is not const, it was not compiling/linking.

Reply:
After trying several combinations I googled the problem and found this: https://stackoverflow.com/questions/185844/how-to-initialize-private-static-members-in-c